### PR TITLE
chore: add asset descriptions alUSD, BUSD, DUSD, FRAX,GUSD,LUSD,MUSD,…

### PR DESCRIPTION
**Pull request by krissnawat**
…TUSD,USDN,USDP

### CODE REVIEW

**Comment by gomesalexandre** [comment]
@krissnawat Thank you for this! Can you resolve the merge conflicts? 🙏🏽

File: packages/asset-service/src/service/descriptions.json [hunk 1]
```diff
@@ -2,11 +2,22 @@
...
...:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0) and [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) is another curve LP token that consists of [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), and [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). To acquire [crvLUSD](/assets/eip155:1/erc20:0xed279fdd11ca84beef15af5d39bb4d4bee23f0ca), deposit [LUSD](/assets/eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0, [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), and/or [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) on [Curve](https://curve.fi/lusd/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg.",
```
Comment (gomesalexandre) [inline packages/asset-service/src/service/descriptions.json:10]: There's a missing closing parenthese resulting in a broken `deposit LUSD` link and missing `DAI` link:
`assets/eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0,%20[DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f`

**Review by gomesalexandre — CHANGES_REQUESTED** [review]
Great job @krissnawat ! One non-working link but otherwise looks good 👍🏽 
The next step after this is fixed will be for @decentralized-j to give the product green light, then this can be merged

File: packages/asset-service/src/service/descriptions.json (hunk 1 above)
Comment (decentralized-j) [reply packages/asset-service/src/service/descriptions.json:10]: once this is fixed then it's good to go from a product standpoint! Thanks @gomesalexandre!

**Comment by decentralized-j** [comment]
@krissnawat after the changes requested are made please send me your ETH address for payment, which will be made on gnossis chain. Thanks!

File: packages/asset-service/src/service/descriptions.json (hunk 1 above)
Comment (krissnawat) [reply packages/asset-service/src/service/descriptions.json]: @gomesalexandre done

**Review by gomesalexandre — APPROVED** [review]
LGTM now! Well done @krissnawat

**Review by theoboldfrazier — APPROVED** [review]
Checked the asset addresses and they match description

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**